backend/wmg/data/ontology_labels.py

```diff
@@ -22,10 +22,10 @@
 
 def ontology_term_label(ontology_term_id: str) -> Optional[str]:
     """
-    Returns the label for an ontology term, given its id. This excludes gene ontology term, which are handled
-    separately by gene_gene_term_label(). Return None if ontology term id is invalid.
+    Returns the label for an ontology term, given its id. This excludes gene ontology term
+    and self reported ethnicity term, which are handled separately by gene_gene_term_label()
```

Comment:
when is gene_term_label used?

Reply:
Right here: https://github.com/chanzuckerberg/single-cell-data-portal/blob/1a8f0992d80e97933072e2262e02712500cea382/backend/wmg/api/v2.py#L339

Reply:
can you also rename this comment to gene_term_label lol